hyp3_metadata/create.py

```diff
@@ -20,6 +19,24 @@ def create_metadata_file_set_rtc(product_dir: Path, granule_name: str, dem_name:
         processor_name=processor_name,
         processor_version=processor_version,
     )
-    writer = RtcMetadataWriter(payload)
+    writer = rtc.RtcMetadataWriter(payload)
 
     return writer.create_metadata_file_set()
+
+
+def create_metadata_file_set_insar(product_dir, granule_name_ref, granule_name_sec, processing_date, looks, dem_name,
```

Comment:
I think I like `ref_granule_name`, `sec_granule_name` better... but I don't feel strongly (just how you typically say it)

`reference_name` I think would also work for me... or `reference_granule_name`...